Dockerfile

```diff
@@ -1,27 +1,46 @@
-FROM golang:1.15.5 as builder
+# Image page: <https://hub.docker.com/_/golang>
+FROM golang:1.15.5-alpine as builder
```

Comment:
I would prefer using just 1.15.5 without alpine. It uses Debian by default and a little big bigger, but from my POV should be more predictable.

Reply:
yep, done